safe/impact_functions/base.py

```diff
@@ -1408,7 +1408,10 @@ def _calculate_impact(self):
         start_time = datetime.now()
 
         # Run the IF. self.run() is defined in each IF.
-        result_layer = self.run()
+        try:
+            result_layer = self.run()
+        except KeywordNotFoundError as e:
+            raise KeywordNotFoundError(e)
```

Comment:
TIL...
So, I was always thinking that I need to raise the same exception if I want to catch it later.
To sum up, if I have already raise certain exception, I don't need to catch it, unless I need it to do more process (e.g. giving error message), is it right?
And the exception will be passed until someone catches it, right?

Let me know I am wrong.

Thanks @Gustry